root/static/scripts/common/components/CDTocLink.js

```diff
@@ -15,10 +15,13 @@ type Props = {
   +cdToc: CDTocT,
   +content: string,
   +subPath?: string,
+  +anchorPath?: string,
```

Comment:
Reorder props using the same
```shell
git diff --name-only master '**.js' | xargs ./script/check_unfixed_eslint_rules --fix
```

```suggestion
  +anchorPath?: string,
  +cdToc: CDTocT,
  +content: string,
  +subPath?: string,
```